### PR TITLE
Add manifesto

### PR DESCRIPTION
**Pull request by zerolab**

### CODE REVIEW

**Review by allcaps — APPROVED** [review]

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+By moving the package to Wagtail Nest, the current maintainer (or original author) agrees to collaborate with the other members and will seek consensus when it comes to contentious changes.
+
+
+## Guidelines
+
+- Be nice
+- The primary goal is maintenance. New features are a nice to have
+- Release often, every time updates are done.
+- Use [SemVer](https://semver.org/). If there are breaking changes, bump the major version.
+- Test against Wagtail main, [nightly](https://github.com/wagtail/wagtail/wiki/Nightly-testing-of-official-plugins).
```
Comment (thibaudcolas) [inline profile/README.md:18]: At first glance it’s not clear to me whether you mean testing against the `main` branch, or the latest nightly release, or both. Could be worth clarifying?

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- Automate the release process
+- Keep project in sync with the Cookiecutter project template
```
Comment (thibaudcolas) [inline profile/README.md:20]: Link?

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+## FAQ
+
+*   What packages are accepted as Wagtail Nest packages, and why?
+    Similar to [Jazzband](https://github.com/jazzband/), we accept packages for the wider community that are in use and in demand.
+    We will allow issues on @wagtail-nest that the community can use to suggest a package to adopt, and the @wagtail-nest members will review.
+    Maintenance takes precedence over active development.
+    
+*   What is the deprecation process?
+    Deprecated packages are moved to https://github.com/orgs/wagtail-nest-deprecated.
```
Comment (thibaudcolas) [inline profile/README.md:36]:
```suggestion
    Deprecated packages are moved to https://github.com/wagtail-nest-deprecated.
```

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+*   Governance: who gets committer/maintainer access?
+    Maintainers can review, merge and release. Maintainers are:
+        * The original author(s) (their own repository).
+        * Anyone who works at companies with core members (all repositories). Vetted people from companies active in the Wagtail community.
+        * Individual member(s) (all repositories)
+            
+    We trust the original author. 
+    We trust the selected companies.
+    You can become a individual member by:
```
Comment (thibaudcolas) [inline profile/README.md:52]:
```suggestion
    You can become an individual member by:
```

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    *   Being a regular contributor
+    *   By being invited by existing maintainers
```
Comment (thibaudcolas) [inline profile/README.md:54]:
```suggestion
    *   Being invited by existing maintainers
```

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    * Anyone can create a pull request to any of the Wagtail Nest repositories. You do not have to be a member to contribute.
+    * Only members can merge and release.
+    * We have the 4 eyes principle. Don’t merge your own pull request.
+
+*   Who manages the Wagtail Nest Github org? Adding and revoking members?
+    In the initial stages, the Wagtail Core team will manage the Nest GitHub org.
+
+*   Who has access to the PyPI keys? Do we need a Wagtail Nest PyPI-account?
+    We do not need a shared Wagtail Nest PyPI account, as the goal is to use scoped tokens that can be added by the PyPI package owners. 
+    We would ask for at least one other maintainer from the agreed list is added to PyPI with the owner role. We will not mandate access for additional maintainers to PyPI as the automated release process should reduce need for it. We believe this is the most secure way to handle releases.
```
Comment (thibaudcolas) [inline profile/README.md:80]: PyPI is very close to releasing their organization features, which will be free for community projects. Once that happens, I think we should mandate transfer of the package over to a `wagtail-nest` organization.

Comment (zerolab) [reply profile/README.md:80]: 👍 we'll revise when that happens

**Pull request merged**